src/palace/manager/integration/license/overdrive/monitor.py

```diff
@@ -65,6 +65,15 @@ def catch_up_from(
         """
         overdrive_data_source = DataSource.lookup(self._db, DataSource.OVERDRIVE)
 
+        if self.collection.parent is None:
+            # If it is a parent account, generate child (Advantage) collections if they don't already exist.
```

Comment:
I'm unsure if we want to do this.   As a note:  we have a CM with a main collection  and a single Advantage account.  However with this block of code included, we create 28 other advantage accounts in the course of an import.  I'm not sure if this is typical and/or a concern, but I wanted to call it out.  
@jonathangreen  - see the ticket for details.